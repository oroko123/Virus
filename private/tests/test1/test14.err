Error in line 15:2000 1990
