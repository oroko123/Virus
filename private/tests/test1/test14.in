znaczek10 200 1990 Polska
znaczek9 100 1995 Grecja
znaczek8 346 1990 Argentyna
znaczek7 850 2000 Maroko
znaczek6 2500 1990 Wielka Brytania
znaczek5 100000 2000 Chiny
znaczek4 13560 1995 Japonia
znaczek3 1285 1990 Holandia
znaczek2 214 1995 Chile
znaczek1 43500 2000 Izrael
1990 1995
1995 2000
1990 2000
1990 1990
2000 1990
