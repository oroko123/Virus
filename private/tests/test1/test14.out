1990 Argentyna 346 znaczek8
1990 Holandia 1285 znaczek3
1990 Polska 200 znaczek10
1990 Wielka Brytania 2500 znaczek6
1995 Chile 214 znaczek2
1995 Grecja 100 znaczek9
1995 Japonia 13560 znaczek4
1995 Chile 214 znaczek2
1995 Grecja 100 znaczek9
1995 Japonia 13560 znaczek4
2000 Chiny 100000 znaczek5
2000 Izrael 43500 znaczek1
2000 Maroko 850 znaczek7
1990 Argentyna 346 znaczek8
1990 Holandia 1285 znaczek3
1990 Polska 200 znaczek10
1990 Wielka Brytania 2500 znaczek6
1995 Chile 214 znaczek2
1995 Grecja 100 znaczek9
1995 Japonia 13560 znaczek4
2000 Chiny 100000 znaczek5
2000 Izrael 43500 znaczek1
2000 Maroko 850 znaczek7
1990 Argentyna 346 znaczek8
1990 Holandia 1285 znaczek3
1990 Polska 200 znaczek10
1990 Wielka Brytania 2500 znaczek6
